src/tests/utils/setup.ts

```diff
@@ -12,13 +12,21 @@ import API from '@presentation/index.js';
 import { beforeAll, afterAll } from 'vitest';
 import type Api from '@presentation/api.interface';
 
+// import { readFileSync } from 'fs';
+
 /**
  * Tests setup maximum duration.
  * Added as default 10000 is not enough
  */
 const TIMEOUT = 200000;
 
 declare global {
+  /**
+   *
+   */
+  /* eslint-disable-next-line no-var */
+  // var truncateTableScript: string;
+
```

Comment:
remove these comments if they are not used

Reply:
removed